system-contracts/scripts/preprocess-bootloader.ts

```diff
@@ -87,19 +87,19 @@ const params = {
     "appendTransactionToCurrentL2Block"
   ),
   RIGHT_PADDED_PUBLISH_TIMESTAMP_DATA_TO_L1_SELECTOR: getPaddedSelector("SystemContext", "publishTimestampDataToL1"),
-  COMPRESSED_BYTECODES_SLOTS: 65536,
+  COMPRESSED_BYTECODES_SLOTS: 196608,
   ENSURE_RETURNED_MAGIC: 1,
   FORBID_ZERO_GAS_PER_PUBDATA: 1,
   KECCAK256_EXPECTED_CODE_HASH: getKeccak256ExpectedHash(),
-  // One of "worst case" scenarios for the number of state diffs in a batch is when 260kb of pubdata is spent
-  // on repeated writes, that are all zeroed out. In this case, the number of diffs is 260kb / 5 = 52k. This means that they will have
-  // accoomdate 14144000 bytes of calldata for the uncompressed state diffs. Adding 260k on top leaves us with
-  // roughly 14404000 bytes needed for calldata. 450125 slots are needed to accommodate this amount of data.
-  // We round up to 451000 slots just in case.
+  // One of "worst case" scenarios for the number of state diffs in a batch is when 780kb of pubdata is spent
+  // on repeated writes, that are all zeroed out. In this case, the number of diffs is 780kb / 5 = 156k. This means that they will have
+  // accoomdate 42432000 bytes of calldata for the uncompressed state diffs. Adding 260k on top leaves us with
```

Comment:
> Adding 260k on top leaves us

This PR plans to have 260k * 3, so either just the comment should be adjusted or the calculation